app/assets/stylesheets/colors.css.scss

```diff
@@ -0,0 +1,10 @@
+// See http://sachagreif.com/sass-color-variables
+
+// Color definitions
+$fsek_orange: #eb7125;
```

Comment:
Name of variable `fsek_orange` should be written in all lowercase letters with hyphens instead of underscores